gadget-container/gadgets/bcck8s/bcc-wrapper.sh

```diff
@@ -119,6 +123,33 @@ if [ "$PROBECLEANUP" = "true" ] ; then
   echo "-:rfd_install" >> /sys/kernel/debug/tracing/kprobe_events 2>/dev/null || true
 fi
 
+if [ "$TOOLMODE" = "default" ] || [ -z "$TOOLMODE" ]; then
+  TOOLMODE="$INSPEKTOR_GADGET_OPTION_DEFAULT_TOOL_MODE"
+fi
+
+if [ "$TOOLMODE" = "auto" ] ; then
+  # kernel provided BTF
+  if test -f /sys/kernel/btf/vmlinux; then
+    TOOLMODE="core"
+  # btfhub / btfgen provided BTF
+  elif test -f /boot/vmlinux-$KERNEL; then
+    TOOLMODE="core"
+  else
+    TOOLMODE="standard"
+  fi
+fi
+
+if [ "$MANAGER" = "false" ] ; then
+  GADGETPATH="$GADGET"
```

Comment:
Using also the `MANAGER` variable to decide the path does not completely convince me. In fact, I think the integration tests are failing because the biotop gadget sets `MANAGER=false` but it does not include the full path in the `--gadget` flag:

https://github.com/kinvolk/inspektor-gadget/blob/df78c4a78f4ce1cb7cd0cf5a709c4202c899e089/cmd/kubectl-gadget/bcck8s.go#L270-L272 

BTW, biotop uses `--nomanager` because it neither needs the `cgroupid`/`mntnsset` maps nor uses the stream mode thus it does not need to register the tracer?

On the other hand, I think we should call the `bcc-wrapper.sh --stop` also with `--nomanager` for the biotop and networkpolicyadvisor (the only ones I found use this option) to avoid trying to delete a trace we know does not exist. If you agree, we could manager it on another PR.